stdlib/3/gc.pyi

```diff
@@ -23,6 +23,9 @@ if sys.version_info >= (3, 8):
 else:
     def get_objects() -> List[Any]: ...
 
+def freeze() -> None: ...
```

Comment:
wasn't sure if we need to guard these with an `if sys.version_info >= (3, 7):` like is already done for `get_objects` or if that is only necessary when there are different types for different versions.

Reply:
looking at CI looks like I should have